src/Common/src/System/IO/PathInternal.Windows.cs

```diff
@@ -53,13 +53,20 @@ internal static partial class PathInternal
 
         internal static readonly char[] InvalidPathChars =
         {
-            '\"', '<', '>', '|', '\0',
```

Comment:
If this is changed, then I believe the function I recently modified (`HasIllegalCharacters`) needs to be changed as well to eliminate checks for these characters.